cockatrice/src/main.cpp

```diff
@@ -197,12 +191,18 @@ int main(int argc, char *argv[])
         app.exec();
     }
 
+    // load custom databased after LoadStatus check, so that they don't bring up the settings dialog
+    if (QDir().mkpath(dataDir + "/customsets"))
+    {
+        // if the dir exists (or has just been created)
+        db->loadCustomCardDatabases(dataDir + "/customsets");
+    } else {
+        qDebug() << "Could not create " + dataDir + "/customsets folder.";
+    }
+
     if (settingsValid()) {
         qDebug("main(): starting main program");
 
-        soundEngine = new SoundEngine;
-        qDebug("main(): SoundEngine constructor finished");
```

Comment:
Yep, i removed it since it was redundant with the messages introduced in a previous PR:
https://github.com/Cockatrice/Cockatrice/blob/master/cockatrice/src/soundengine.cpp#L155
https://github.com/Cockatrice/Cockatrice/blob/master/cockatrice/src/soundengine.cpp#L46